installation-manager/src/main/java/org/wildfly/core/instmgr/logging/InstMgrLogger.java

```diff
@@ -81,6 +86,18 @@ public interface InstMgrLogger extends BasicLogger {
     @Message(id = 20, value = "No custom patches installed found for the specified manifest maven coordinates: '%s'")
     OperationFailedException noCustomPatchFound(String manifestGA);
 
+    @LogMessage(level = Logger.Level.INFO)
+    @Message(id = 21, value = "Installation provisioning is using the following channels: '%s'")
```

Comment:
Nitpick, but I wonder if this should read something like "Installation provisioned using the following channel states/(versions?):". The ManifestVersions provide information about the latest used versions from each channel rather then channel information like manifest location, repositories etc.

Reply:
Thanks @spyrkob -- good point. I went with "Installation was provisioned using the following channel versions:".